### PR TITLE
VerifiablePresentationManager implementation

### PR DESCRIPTION
**Pull request by lucmir**
Implements all the interface defined for `VerifiablePresentationManager`, except for the `wasGrantedForDSR` method (it requires a fix in the Credential issuing to fill the `grantedFor` property).

The interface for Credentials and Evidences are defined in `src/Credential.ts`. `PresentationVerifier` holds the methods to verify a Credential (it uses methods from `@identity.com/credential-commons`).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,8 +9,10 @@
   "scripts": {
     "format": "prettier --write \"src/**/*.ts\" \"src/**/*.js\"",
     "lint": "tslint -p tsconfig.json",
+    "lint:fix": "tslint -p tsconfig.json --fix",
     "build": "tsc",
-    "test": "jest --config jestconfig.json",
+    "test": "jest --config jestconfig.json --coverage",
+    "test:file": "jest --config jestconfig.json --",
```
Comment (jpsantosbh) [inline package.json]: this trailing -- is correct?

File: src/VerifiablePresentationManager.ts [hunk 2]
```diff
@@ -243,16 +349,145 @@ export class VerifiablePresentationManager {
...
+                throw new Error(`Unverified Presentations: ${_.join(unverifiedIds)}`);
+            }
+        }
+        return verifiedPresentations;
+    }
 
+    private verifyEvidences(verifiedPresentations : Credential[], notThrow = this.options.notThrow) : Evidence[] {
+        const verifiedEvidences : Evidence[] = [];
+        this.artifacts.evidences.forEach(evidence => {
+            // TODO check sha256
```
Comment (jpsantosbh) [inline src/VerifiablePresentationManager.ts]: this need to be implemented

File: package.json (hunk 1 above)
Comment (lucmir) [reply package.json]: Yes. I am using this to run a single test file from the terminal:
```
 npm run test:file src/__tests__/index.test.ts
```

File: src/VerifiablePresentationManager.ts (hunk 2 above)
Comment (lucmir) [reply src/VerifiablePresentationManager.ts]: Done

**Pull request merged**